rootfs/ingest/python/qserv/ingest.py

```diff
@@ -210,49 +251,42 @@ def _get_chunk_location(repl_url, chunk, database, transaction_id):
 
     # Get location host and port
     host = responseJson["location"]["host"]
-    port = responseJson["location"]["port"]
+    # FIXME Ask Igor Gaponenko for a method to get this parameter?
+    # Or set it from qserv-operator?
+    port = 25004
```

Comment:
The documentation on this service https://confluence.lsstcorp.org/pages/viewpage.action?pageId=135955129 says that you should use:
```
host = responseJson["location"]["http_host"]
port = responseJson["location"]["http_port"]
```

Reply:
:+1: